### PR TITLE
added support for the version and listing important file paths

### PR DESCRIPTION
**Pull request by omkarkhatavkar**
this resolves : https://github.com/RedHatQE/cloudwash/issues/46
```
> swach --version
Version: 0.0.0
Settings File: /Users/okhatavk/satellite/cloudwash/broker_settings.yaml
```

### CODE REVIEW

**Comment by jyejare** [comment]
@omkarkhatavkar Does that still shows 0.0.0 as you have shown in the description of this PR.

**Comment by omkarkhatavkar** [comment]
```
> swach --version
Version: 1.0.2
Settings File: /Users/okhatavk/satellite/cloudwash/broker_settings.yaml

```

File: cloudwash/config.py [hunk 1]
```diff
@@ -1,13 +1,18 @@
+from pathlib import Path
+
 from dynaconf import Dynaconf
 from dynaconf import Validator
 
 from cloudwash.logger import logger
 
+CURRENT_DIRECTORY = BROKER_DIRECTORY = Path()
```
Comment (jyejare) [inline cloudwash/config.py:8]:
```suggestion
CURRENT_DIRECTORY = BROKER_DIRECTORY = PurePath()
```

File: cloudwash/config.py [hunk 2]
```diff
@@ -1,13 +1,18 @@
+from pathlib import Path
```
[the remark this answers is not in the document]
Comment (omkarkhatavkar) [reply cloudwash/config.py:1]: the absolute path I needed to show in the version output.

**Review by jyejare — APPROVED** [review]

**Pull request merged**